tests/Unit/LoggerTraitTest.php

```
@@ -0,0 +1,230 @@
+
+  /**
+   * Test logSection method when verbose mode is disabled.
+   */
+  public function testLogSectionWithVerboseDisabled(): void {
+    static::loggerSetVerbose(FALSE);
+
+    // This should not output anything and not throw exceptions.
+    static::logSection('TEST TITLE', 'Test message');
+
+    // Verify method executed silently without exceptions.
+    $this->assertTrue(TRUE);
```

PHPStan (2.1.15)

Call to method PHPUnit\Framework\Assert::assertTrue() with true will always evaluate to true.


(method.alreadyNarrowedType)